rasa/core/policies/ensemble.py

```diff
@@ -60,6 +62,14 @@ def _training_events_from_trackers(training_trackers):
 
         return events_metadata
 
+    @staticmethod
+    def check_missing_policies(ensemble, domain) -> None:
+        """Check for domain elements that work only with certain policies."""
+
+        policies_needing_validation = [FormPolicy, MappingPolicy]
```

Comment:
there is also a check in the twostagefallback policies for the `out_of_scope` event. Could we also move it here?

Reply:
Could you point me to this? I can't seem to find it.